specs/agents/log-onboarding.md

```diff
@@ -37,7 +37,19 @@ app as ECS-compatible json, as shown in the
 [spec](https://github.com/elastic/ecs-logging/blob/master/spec/spec.json).
 
 For all options other than `off`, the [log correlation](log-correlation.md) should be implicitly enabled.
-### Required fields
+
+## `log_ecs_formatter_allow_list` configuration
+
+Only formatters that match an item on this list will be automatically reformatted to ECS when `log_ecs_reformatting` is set to any option other than `OFF`. A "formatter" is a generic name used to describe the logging-framework-specific entity that is responsible for the formatting of log events. For example, in some Java frameworks those would be subtypes of `Layout` or `Encoder`.
```

Comment:
```suggestion
Only formatters that match an item on this list will be automatically reformatted to ECS when `log_ecs_formatting` is set to any option other than `off`. A "formatter" is a generic name used to describe the logging-framework-specific entity that is responsible for the formatting of log events. For example, in some Java frameworks those would be subtypes of `Layout` or `Encoder`.
```

Should this be log_ecs_formatting in line with the spec (and renamed to log_ecs_reformatting in a separate PR is that makes sense)?

Reply:
Sure, let's do it this way